src/remote.cc

```diff
@@ -682,6 +688,102 @@ static void gr_slideshow_delay(const gchar *text, GIOChannel *, gpointer)
 	options->slideshow.delay = static_cast<gint>(n * 10.0 + 0.01);
 }
 
+static void gr_duplicates_threshold(const gchar *text, GIOChannel *, gpointer)
+{
+	gint thresh;
+	gint n;
+	gint res;
+
+	res = sscanf(text, "%d", &thresh);
+	if (res == 1)
+		{
+		n = thresh;
+		if (n < 0 || n > 100)
+			{
+			printf_term(TRUE, "Image similarity threshold out of range (%d to %d)\n", 0, 100);
+			return;
+			}
+		}
+	else
+		{
+		n = 99;
+		}
+
+	options->duplicates_similarity_threshold = static_cast<guint>(n);
```

Comment:
Looks like `n` is redundant. If you'd like to have intermediate variable consider changing its type to `guint`.